bin/write-mappings.js

```diff
@@ -7,18 +7,22 @@
 
 const fs = require('fs')
```

Comment:
No need to require individual factories piecemeal anymore.
Now that NyplCoreObjects falls back on default parsing - we can go in through it's front door.

This file has to deal with the fact that I've been pushing underscoreized files to S3. But NyplCoreObjects's front-door looks for dasherized strings.